AK/Optional.h

```diff
@@ -63,6 +66,24 @@ class [[nodiscard]] Optional {
         }
     }
 
+    template<typename U>
+    requires(IsConstructible<T, U const&> && !IsSpecializationOf<T, Optional> && !IsSpecializationOf<U, Optional>) ALWAYS_INLINE explicit Optional(Optional<U> const& other)
+        : m_has_value(other.m_has_value)
+    {
+        if (other.has_value()) {
+            new (&m_storage) T(other.value());
+        }
```

Comment:
curly braces literally *useless* here

Reply:
They are indeed, I just copied it from the constructor above, should I fix it up as well? :^)

Reply:
might as well, yeah :)